rules/linux/lnx_system_network_connections_discovery.yml

```diff
@@ -0,0 +1,26 @@
+title: System Network Connections Discovery
+id: 4c519226-f0cd-4471-bd2f-6fbb2bb68a79
+status: experimental
+description: Detects usage of system utilities to discover system network connections
+author: Daniil Yugoslavskiy, oscd.community
+date: 2020/10/19
+references:
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1049/T1049.md
+logsource:
+  category: process_creation
+  product: linux
+detection:
+  selection:
+    Image|endswith: 
+      - '/who'
+      - '/w'
+      - '/last'
```

Comment:
The last three are often used in administration for debugging purposes and would cause lot of FPs.

Reply:
discussion here https://github.com/SigmaHQ/sigma/pull/1362#discussion_r598336458